examples/n2hp_cube_example.py

```diff
@@ -32,6 +33,8 @@
 else:
     # Run the fitter
     # Estimated time to completion ~ 2 minutes
+    spc.xarr.refX = 23*u.Hz
+    spc.xarr.velocity_convention = 'radio'
```

Comment:
actually, maybe it does.  Somewhere in the n2hp file, there should be a rest frequency - use that.